packages/vite/src/node/server/openBrowser.ts

```diff
@@ -91,7 +91,7 @@ function startBrowserProcess(browser: string | undefined, url: string) {
   // Fallback to open
   // (It will always open new tab)
   try {
-    const options = { app: browser, url: true }
+    const options: open.Options = { app: { name: browser! } }
```

Comment:
`browser` can be undefined, so I think we need something like this here:
```suggestion
    const options: open.Options = browser ? { app: { name: browser } } : {}
```

Reply:
applied your suggestion but I had tested by setting browser to undefined and it worked fine